src/base/dof_map.C

```diff
@@ -2322,8 +2322,7 @@ void DofMap::SCALAR_dof_indices (std::vector<dof_id_type> & di,
 bool DofMap::semilocal_index (dof_id_type dof_index) const
 {
   // If it's not in the local indices
-  if (dof_index < this->first_dof() ||
-      dof_index >= this->end_dof())
+  if (!this->local_index(dof_index))
```

Comment:
There's a dozen other places we could make this change too, while we're at it.

Reply:
I did at least admit my laziness in the commit log. :) I'll do a quick grep later to see if I can find a few more.